### PR TITLE
Fixed #311 Added styles to navbar

### LINKED ISSUE #311: Harmonise the logo and the top bar
**I'm submitting a ...**  (check one with "x")
- [x] feature request
channel at https://gitter.im/fossasia/meilix

**Actual Behaviour**
The logo is a colourful rectangle. It would be useful to also change the background colour and the text colour of the top bar.

**Would you like to work on the issue?**
No. Only for Google Code In participants.

### PR DESCRIPTION
**Pull request by STEdirisuriya**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] My branch is up-to-date with the Upstream `master` branch.

#### Short description of what this resolves:
I fixed issue #311. I added some styles to navbar to harmonise the logo and the top bar as the issue. 
image

And I added a hover effect.
image

#### Changes proposed in this pull request:

- Change the color of nav bar.
- Add a hover effect to the navbar.

Fixes #311 

Thank you!
@tabesin Can you have a look?

### CODE REVIEW

**Comment by sarnava1** [comment]
@STEdirisuriya Thank you for the PR. I see you have checked all the tickboxes. Can you tell me what tests  you have carried out. If you have not carried out any test then please do not tick those boxes from the next time. Also it would have been good if you had given a preview link. Anyways LGTM @tabesin @mariobehling @meets2tarun .

**Comment by STEdirisuriya** [comment]
> @STEdirisuriya Thank you for the PR. I see you have checked all the tickboxes. Can you tell me what tests you have carried out. If you have not carried out any test then please do not tick those boxes from the next time. Also it would have been good if you had given a preview link. Anyways LGTM @tabesin @mariobehling @meets2tarun .

@sarnava1 sorry for my mistake. I removed it now. Thank you!

**Comment by tabesin** [comment]
You should then also change the color of the black elements to white.

**Comment by STEdirisuriya** [comment]
image
@tabesin I changed it. Thanks.

File: static/css/style.css [hunk 1]
```diff
@@ -33,14 +33,15 @@ body {
...
+  background-color: #a60041;
 }
 
 .custom-navtitle {
   display: inline-block;
   vertical-align: middle;
   margin: 10px;
   font-size: 30px;
-  color: black;
+  color: white;
```
Comment (STEdirisuriya) [inline static/css/style.css]: Here.

File: static/css/style.css [hunk 2]
```diff
@@ -67,6 +68,10 @@ h3.custom-navtitle {
   background-color:#F0EEEE;
 }
 
+.custom-menubutton i{
+  color: white;
```
Comment (STEdirisuriya) [inline static/css/style.css]: And here.

**Comment by abishekvashok** [comment]
Task link: https://codein.withgoogle.com/dashboard/task-instances/4663115040948224/

**Pull request merged**